spec/integration/outputs/retry_spec.rb

```diff
@@ -78,7 +88,7 @@ def mock_actions_with_response(*resp)
     subject.receive(event1)
     subject.receive(event2)
     subject.buffer_flush(:final => true)
-    sleep(3)
+    sleep(10)
```

Comment:
I remember we set this to `10` during the weird test session. we can return this to `3`